### PR TITLE
Store location changes

### PR DESCRIPTION
**Pull request by ManoharVanam**
Merged Ashok and My changes from the version

### CODE REVIEW

File: core/src/main/java/org/carbondata/core/carbon/metadata/converter/SchemaConverter.java [hunk 1]
```diff
@@ -101,5 +101,5 @@ TableSchema fromExternalToWrapperTableSchema(
    * @return
    */
   TableInfo fromExternalToWrapperTableInfo(org.carbondata.format.TableInfo externalTableInfo,
-      String dbName, String tableName);
+      String dbName, String tableName, String storePath);
```
Comment (Vimal-Das) [inline core/src/main/java/org/carbondata/core/carbon/metadata/converter/SchemaConverter.java]: Use consistent name "storePath" or "storeLocation" in all classes.

Comment (ravipesala) [inline core/src/main/java/org/carbondata/core/carbon/metadata/converter/SchemaConverter.java]: @Vimal-Das I have basic question. Why we require storePath in thrift schema?

**Comment by ManoharVanam** [comment]
@Vimal-Das  fixed all review comments, pls review and merge

**Comment by Vimal-Das** [comment]
LGTM

**Comment by Vimal-Das** [comment]
Thanks for working on this, merged to master.

**Pull request merged**